build.d/300-oca-dependencies

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import yaml
+import logging
+import urllib2
+from glob import iglob
+
+from odoobaselib import (
+    CUSTOM_DIR, AUTO_ADDONS_YAML, AUTO_REPOS_YAML, ADDONS_YAML, REPOS_YAML
+)
+
+# Skip aggregation of dependencies if AGGREGATE env var is not set
+if not os.environ.get('AGGREGATE'):
+    logging.warning("Skipping oca dependency aggregation")
+    sys.exit()
+
+BASE_URL = 'https://raw.githubusercontent.com/OCA'
+
+
+# Method from maintainer-quality-tools
+def parse_depfile(depfile, owner='OCA'):
+    deps = set()
+    for line in depfile:
+        line = line.strip()
+        if not line or line.startswith('#'):
+            continue
+        parts = line.split()
+        repo = parts[0]
+        if len(parts) > 2:
+            branch = parts[2]
+        else:
+            branch = os.environ['ODOO_VERSION']
+        if len(parts) > 1:
+            url = parts[1]
+        else:
+            url = 'https://github.com/%s/%s.git' % (owner, repo)
```

Comment:
You should add to the dockerfile `ONBUILD ARG DEFAULT_REPO_PATTERN="https://github.com/OCA/%s.git"` and rely on that env variable instead.